### ISSUE TITLE
Insert a chart to the web page

### ISSUE THREAD

**Issue by reflash** [TahaHachana/XPlot#40]
Hello,

I've recently found this beautiful lib and tried it for my needs. 
I've faced several problems working with XPlot and currently could not find how to display chart on a web page.
I use a https://suave.io to start a web server and the following code
```fsharp
open Suave
let html = chart.GetInlineHtml ()
startWebServer defaultConfig (Successful.OK html )
```

The problem is that if I do so, then I have no js that is responsible for Plotly display in header, so it fails to render on client site. 
Is there any way I could use the chart html like this? Or this way is completely not the way of how this should be used?

**Comment by TahaHachana** [comment]
Hello,

There is a `GetHtml` method that returns a full HTML document that you can serve as shown above. `GetInlineHtml` would be useful if you want to generate the document by yourself and display additional content besides the chart.

**Comment by reflash** [comment]
Thank you, I tried it with Plotly chart, but found only GetInlineHtml method. 
As I saw, there's something like a builder pattern, so I can maybe build the html parts, such as html header with appropriate script file and so on.
In my case I really need InlineHtml, so my question is how can I add a Plotly lib to header and is there a method for this case?

**Comment by TahaHachana** [comment]
Actually the `GetHtml` method for Plotly charts will be available in the next release. For now you just need to insert the inline markup in a simple document like this one https://github.com/TahaHachana/XPlot/blob/master/src/XPlot.Plotly/Main.fs#L9.

**Comment by TahaHachana** [comment]
The `GetHtml` method is now available for Plotly charts too in version 1.4.2.
